src/store/index.ts

```diff
@@ -50,6 +57,9 @@ export default createStore<State>({
       uploadResult: { accessKey: string; uuid: Uint8Array }
     ) {
       state.uploadResult = uploadResult;
+    },
+    setDecryptedMsg(state, msg) {
```

Comment:
(As above, with some type signatures)

```suggestion
    setExecutionToken(state: State, executionToken: Uint8Array) {
```